docs/sdkx-cocos2dx/sdk-configuration.mdx

```diff
@@ -0,0 +1,486 @@
+
+    ValueMap config;
+    config["enableInAppNotification"] = false;
+
+    HelpshiftCocos2dx::install( "<App Id from the Helpshift Dashboard>",
+                                "<Domain name from the Helpshift Dashboard>",
+                                config);
+    ...
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Initiate new conversation on chat screen load {#initiate-chat-on-load}
+
+In `config` map of `HelpshiftCocos2dx::showConversation(config);` API, setting the `initiateChatOnLoad` option to `true` will start a new conversation if the previous issue is resolved, without the end user clicking the **New Conversation** button or going through the post resolution flows like feedback bots of the previous issue.
+
+|               |                                        |
+| ------------- | -------------------------------------- |
+| Option:       | <code>initiateChatOnLoad</code>        |
+| Values:       | <code>true</code> / <code>false</code> |
+| Default:      | <code>false</code>                     |
+| Min SDK       | v10.2.0                                |
+| Supported by: | <code>showConversation</code>          |
+
+Example:
+
+```cpp
+cocos2d::ValueMap chatConfig;
+chatConfig["initiateChatOnLoad"] = true;
+HelpshiftCocos2dx::showConversation(chatConfig);
+```
+
+## Android specific configurations
+
+### Screen Orientation {#screen-orientation}
+
+The screen orientation of Helpshift SDK screens can be fixed by setting the `screenOrientation` to constants available in the [ActivityInfo](https://developer.android.com/reference/android/content/pm/ActivityInfo.html) class.
+For example, you may want to fix the orientation to `ActivityInfo.SCREEN_ORIENTATION_PORTRAIT` for mobile users and `ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE` for tablet users.
+
+<div className="compact ">
+
+|         |                                                                                                                                        |
+| ------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Flag    | screenOrientation                                                                                                                      |
+| values  | integer values for orientation from [ActivityInfo](https://developer.android.com/reference/android/content/pm/ActivityInfo.html) class |
+| default | ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED                                                                                            |
+
+</div>
+
+Example:
+
+```java
+import com.helpshift.HelpshiftCocosBridge;
+
+public class MyActivity extends Cocos2dxActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Add screenOrientation config in install config map
+        Map<String, Object> config = new HashMap<>();
+        config.put("screenOrientation", 0);
```

Comment:
Since this is java code, we can use ActivityInfo class for orientation constants